controller_manager/scripts/spawner

```diff
@@ -88,22 +81,23 @@ def parse_args(args=None):
                         help='namespace of the controller_manager services')
     parser.add_argument('--timeout', metavar='T', type=float, default=30,
                         help='how long to wait for controller_manager services when starting up [s] (default: 30)')
-    parser.add_argument('--shutdown-timeout', metavar='T', type=float, default=30,
```

Comment:
I guess at least ` metavar='T', type=float` should be kept. 
Haven't tested it yet myself.

Reply:
I've tested it, works fine with the defaults.

[metavar](https://docs.python.org/3/library/argparse.html#metavar):  "By default, ArgumentParser objects use the dest value as the “name” of each object. "
[type](https://docs.python.org/3/library/argparse.html#type): "By default, ArgumentParser objects read command-line arguments in as simple strings."